pkg/flare/archive.go

```diff
@@ -287,8 +287,21 @@ func writeStatusFile(tempDir, hostname string, data []byte) error {
 	return err
 }
 
+func addParentPerms(dirPath string, permsInfos permissionsInfos) {
+	parent := filepath.Dir(dirPath)
+	for parent != "." {
```

Comment:
Is `parent != "."` required?

Reply:
`filepath.Dir` returns `"."` when the path is empty, which may occur.